src/lib/FreeC/Backend/Agda/Syntax.hs

```diff
@@ -29,17 +39,21 @@ import           Agda.Syntax.Position
 
 -- | Creates a (not qualified) Agda variable name from a 'String'.
 name :: String -> Name
-name ident = Name NoRange InScope [Id ident]
+name str = Name NoRange InScope [Id str]
 
 -- | Create a qualified identifier given a local identifier as 'Name' and a
 --   list of module 'Name's.
 qname :: [Name] -> Name -> QName
-qname modules ident = foldr Qual (QName ident) modules
+qname modules n = foldr Qual (QName n) modules
```

Comment:
I have the feeling that we usually use `n` for `Interger` values. Perhaps we can use `name` as variable name, if we rename the function `name` to `varName` or something similar?

Reply:
`name` would be a good name, but it produces a show warning, because `name :: String -> Agda.Name` is defined in the same module. `varName` is problematic, because `Agda.QName`s are used for modules and functions as well. I will try to finde a better name.